CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to the Conjur Katacoda Scenarios
+
+For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Katacoda Documentation](#katacoda-documentation)
+- [Building and Testing](#building-and-testing)
+- [Troubleshooting Steps](#troubleshooting-steps)
+- [Contribution Workflow](#contribution-workflow)
+
+## Prerequisites
+The Katacoda CLI is recommended in order to create a new scenario. You can install the Katacoda CLI by running `npm install katacoda-cli --global` in the terminal.
+
+## Katacoda Documentation
+A link to the Katacoda documentation can be found [here](https://www.katacoda.community/welcome.html).
+
+Additionally, it is recommended that you look at the other scenarios in the `cyberark/katacoda-scenarios` to see how Katacoda has been used in other tutorials.
+
+## Building and Testing
+The Katacoda scenarios can be tested using the `katacoda-cli`. In order to validate that the structure of your scenario is properly defined, you can run `katacoda validate:all .` in the root directory of the `cyberark/katacoda-scenarios` directory.
+
+In order for your scenario to be properly validated, all extraneous files (if any) must be included directly in the `assets` directory of your scenario.
+
+Our repository also runs this command automatically using a Github action.
+
+## Troubleshooting Steps
+Katacoda may take some time to properly load in changes to your scenario. It can take anywhere from 5 to 30 minutes for the scenario to propagate onto Katacoda.
+
+If it takes longer than this to run, we highly recommend checking your Github log and pushing again if necessary.
+
+## Contribution Workflow
+1. Fork the repository to your Github account (this is necessary in order to set up a [Katacoda webhook](https://katacoda.com/teach/git-hosted-scenarios)
+2. If issue for relevant change has not been created, open one [here](https://github.com/cyberark/katacoda-scenarios/issues)
+3. Add the `implementing` label to the issue once you begin to work on it
+4. Commit your changes (`git commit -m 'Add X feature that does Y'`)
```

Comment:
Using `-m` in this way omits the commit description - we shouldn't use in this manner. You can probably just leave `git commit` by itself here.